app/views/posts/index.html.erb

```diff
@@ -1 +1,14 @@
 <h2>Here is a list of posts for a given user<h2>
+<%= render partial: "users/user" , locals: { user: @user } %>
+ 
+<div class="post-container">
+  <% @posts.each_with_index do |post, index| %>
+    <%= render partial: "posts/post" , locals: { post: post, index: index } %>
+
+      <ul>
+        <% post.comments.each do |comment| %>
+          <% end %>
+      </ul>
```

Comment:
- [ ] It would be better if we are able to display 5 most recent comments in our post/index.html.erb file. Kindly check the example below

```
<% post.most_recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>

```